docker/dockerfiles/Dockerfile

```diff
@@ -106,6 +106,12 @@ RUN apt update && \
 
 RUN mkdir -p ${INTEL_OPENVINO_DIR}
 
+RUN curl ${PACKAGE_LINK} -o /tmp/package.tgz \
+    && mkdir -p /tmp/openvino/ \
+    && tar -xzf /tmp/package.tgz -C /tmp/openvino/ \
+    && rm -rf /tmp/package.tgz \
+    && DIRNAME=$(ls /tmp/openvino/) && mv -v /tmp/openvino/${DIRNAME}/* ${INTEL_OPENVINO_DIR}
```

Comment:
Let's try to move package downloading to the particular stage in deployment pipeline